packages/create/bin/create-stacks-workspace.ts

```diff
@@ -155,12 +156,14 @@ async function getConfiguration(argv: yargs.Arguments<CreateStacksArguments>) {
 async function main(parsedArgv: yargs.Arguments<CreateStacksArguments>) {
     const { nxVersion, ...forwardArgv } = parsedArgv;
     const { name } = forwardArgv;
-    const argumentsToForward = unparse(forwardArgv as unparse.Arguments, {
-        alias: {
-            packageManager: ['pm'],
-            interactive: ['i'],
-        },
-    });
+    const argumentsToForward = replaceNextPreset(
```

Comment:
is it easier to modify the forwardArgv as an object, before unparsing? We can probably rename `replaceNextPreset` to something like `normaliseForwardedArgv`

Reply:
I have updated the naming. Don't quite get your point about parsing? I understand the unparse method is already modifying the object into appropriate format we require here...